src/formGenerator.vue

```diff
@@ -342,6 +357,11 @@ div.vue-form-generator(v-if='schema != null')
 				return res.map(item => item.error);
 			},
 
+			fieldTouched(field) {
+				let res = this.touched.filter(e => e.field == field);
```

Comment:
you could use
`let field = this.touched.find(e=>e.field == field)`
which returns null or the first match. This way you don't have to use `res.map(item => item.touched)[0];` which is not very safe